crypto/rand/rand_lib.c

```diff
@@ -40,14 +41,14 @@ static CRYPTO_ONCE rand_init = CRYPTO_ONCE_STATIC_INIT;
  * it's not sufficient to indicate whether or not the seeding was
  * done.
  */
-void rand_rdtsc(void)
+void rand_read_tsc(RAND_poll_fn cb, void *arg)
 {
     unsigned char c;
     int i;
 
     for (i = 0; i < 10; i++) {
```

Comment:
Ten bytes is probably excessive here, especially when credited as forty bits.
Maybe four bytes instead.